inspire/base/searchext/mappings/jobshidden.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
Do we need to separate jobshidden and regular jobs?

Hidden jobs are simply expired jobs, but they have the same schema. So we could simply solve this by using native invenio collections? (we could simply restrict hidden jobs).

@jalavik, @jmartinm ?